build/player_entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/zsh
+
+if [[ ! $(whoami) == "player" ]] 
+	then echo "I'm not the player"; exit 1; 
+fi
+
+# https://git-scm.com/book/en/v2/Git-on-the-Server-Setting-Up-the-Server
+cd
+pwd
+ssh-keygen -q -t rsa -N '' -f ~/.ssh/id_rsa 2>/dev/null <<< y >/dev/null
+
+cat ~/.ssh/id_rsa.pub >> /tmp/id_rsa.player.pub
+
+echo "Setting up zsh"
+sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
+echo "ZSH_THEME=\"agnoster\"" >> ~/.zshrc
```

Comment:
Maybe something without powerline symbols, but still with good git support?
`juanghurtado` for example. https://github.com/ohmyzsh/ohmyzsh/wiki/Themes

Reply:
Fixed